src/scenes/CreateEvent/index.js

```diff
@@ -1,28 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 import _ from 'lodash';
 import moment from 'moment';
 import Web3Utils from 'web3-utils';
-import { Field, FieldArray, reduxForm, Form, formValueSelector, change, reset } from 'redux-form';
+import { Field, reduxForm, Form, formValueSelector, change, reset } from 'redux-form';
 import Dialog, { DialogActions, DialogContent, DialogTitle } from 'material-ui/Dialog';
 import Grid from 'material-ui/Grid';
-import TextField from 'material-ui/TextField';
-import Input, { InputLabel, InputAdornment } from 'material-ui/Input';
+import _TextField from 'material-ui/TextField';
```

Comment:
Yeah! If you look at line 445, it's just because then we can use the same name. We're making our own stateless component to essentially accomplish the same goal.